test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -143,6 +143,10 @@
     ("aten::new_empty.SymInt_out", datetime.date(2022, 11, 30)),
     ("aten::zeros.SymInt", datetime.date(2022, 11, 30)),
     ("aten::zeros.SymInt_out", datetime.date(2022, 11, 30)),
+    ("aten::amax", datetime.date(2022, 9, 15)),
```

Comment:
note that if you want to reduce the merge conflicts on this file, you can add your changes in the middle of it ;)

Reply:
Oh good to know, I'll do that in the future